ci/prow/config_knative.yaml

```diff
@@ -177,6 +177,12 @@ presubmits:
     - unit-tests: true
       dot-dev: true
 
+  knative/sample-source:
```

Comment:
Please also add CI jobs for it, as what we have for `sample-controller` - https://github.com/knative/test-infra/blob/2c4cd9df33c648a86d95bdd4119386cba04cde56/ci/prow/config_knative.yaml#L416.

It does not hurt for not having it though, but it's better to follow the same pattern.

Reply:
Ah sorry, thought that might've been what was meant by the guide saying to not setup presubmit tests just yet.  Will add it now.

Reply:
added in a80e114